MaterialSkin/HTML/material/html/js/utils.js

```diff
@@ -1041,7 +1041,8 @@ function getClickPos(ev) {
 }
 
 function useArtistTagType(genre, genres) {
-    return (genre && genres.has(genre)) || (1==genres.size && genres.has('*'));
+    const genreArray = genre.toString().split(/\s*,\s*/);
```

Comment:
Don't like this converting to array each and every time a tag type is checked. You should change the `splitMultiples(i);` call @line 103 in `queue-page.js` to ` splitMultiples(i, true);` - this _should_ then convert `genres` into an array. This `useArtistTagType` could then be:

```javascript
function useArtistTagType(genres, tagGenres) {
        return (1==tagGenres.size && tagGenres.has('*')) || (genres.some(g => tagGenres.has(g)));
}
```